app/models/shipit/pull_request.rb

```diff
@@ -108,6 +112,7 @@ def reject!(reason)
     def reject_unless_mergeable!
       return reject!('merge_conflict') if merge_conflict?
       return reject!('ci_failing') unless all_status_checks_passed?
+      return reject!('timedout') if timedout?
```

Comment:
For my clarification: by having a minutely cron job, this will run not long after after the hour is over. Right?